packages/prop-types/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@pluralsight/ps-design-system-prop-types",
```

Comment:
creating a new `prop-types` for custom prop types like `elementOfType` and possible future prop types